### PR TITLE
Feature/init option

### PR DESCRIPTION
**Pull request by jawira**
Please see: phingofficial/phing#796

### CODE REVIEW

File: classes/phing/Phing.php [hunk 1]
```diff
@@ -70,6 +70,7 @@ class Phing
 
     /** The default build file name */
     const DEFAULT_BUILD_FILENAME = "build.xml";
+    const DEFAULT_PROPERTIES_FILENAME = "build.properties";
```
Comment (siad007) [inline classes/phing/Phing.php]: Do we really need a build.properties file?

File: classes/phing/Phing.php [hunk 2]
```diff
@@ -1048,6 +1055,56 @@ public static function printVersion()
...
+    }
+
+    /**
+     * Creates generic Property file
+     *
+     * @param string $dir
+     */
+    protected static function initPropertyFile($dir)
+    {
+        $content = '[section]' . PHP_EOL;
```
Comment (siad007) [inline classes/phing/Phing.php]: Sections are not supported by property files but by ini files.

**Comment by jawira** [comment]
Changes have been made :)

File: classes/phing/Phing.php [hunk 3]
```diff
@@ -1048,6 +1055,38 @@ public static function printVersion()
     }
 
     /**
+     * Creates generic buildfile
+     */
+    public static function init()
+    {
+        $homeDir = self::getProperty(self::PHING_HOME);
```
Comment (siad007) [inline classes/phing/Phing.php]: I think we could take the app start directory here `self::getProperty('application.startdir')`

**Comment by jawira** [comment]
Regarding the default directory, my intention was to create de sample build.xml in the current directory as you mention. I will correct this.

About overriding default location and/or file name, I wanted to keep it simple but I agree it's a nice feature to have. I will start working on this feature.

**Comment by jawira** [comment]
Hello guys, now -init option accepts an optional path, it can be a directory or a file name.
Please let me know if you have any question. I'm waiting for your feedback too :)

**Comment by mrook** [comment]
Looks good! Except for two last things. One very minor, it looks like some accidental white space changes slipped into the PR. Lastly, would it be nice to throw an exception when the (specified) build file already exists? Rather than silently stopping.

**Comment by jawira** [comment]
Hello, thanks for your comments. I have done the requested changes :)

**Pull request merged**